proto/poktroll/migration/tx.proto

```diff
@@ -111,3 +114,54 @@ message MsgClaimMorseAccountResponse {
   int64 session_end_height = 3 [(gogoproto.jsontag) = "session_end_height"];
 }
 
+// TODO_UPNEXT(@bryanchriswhite, #1034):
+// - Refactor to consistent naming convention
+
+// MsgClaimMorseApplication is used to execute a claim (one-time minting of tokens on Shannon),
+// of the total tokens owned by the given Morse account, according to the on-chain MorseClaimableAccounts,
+// to the balance of the given Shannon account, followed by staking that Shannon account as an application.
+message MsgClaimMorseApplication {
+  option (cosmos.msg.v1.signer) = "shannonDestAddress";
+
+  // The bech32-encoded address of the Shannon account to which the claimed tokens
+  // will be minted and from which the application will be staked.
+  string shannonDestAddress = 1;
```

Comment:
NIT (sorry for not catching it earlier).

Can we make this snake case to comply with best practices of protos elsewhere?

It'll still be camel case when converted to a struct.

Reply:
i.e. shannon_dest_address

Reply:
This is done in #1082.